### PR TITLE
Minor docs fixes

### PR DESCRIPTION
**Pull request by chbatey**

### CODE REVIEW

File: docs/src/main/paradox/client.md [hunk 1]
```diff
@@ -20,7 +20,7 @@ enablePlugins(AkkaGrpcPlugin)
 @@@
 
 Gradle
-:   @@@vars
+:  @@@vars
```
Comment (chbatey) [inline docs/src/main/paradox/client.md]: this was broken

File: plugin-tester-scala/src/main/scala/io/grpc/examples/helloworld/GreeterServer.scala [hunk 2]
```diff
@@ -35,7 +35,8 @@ object GreeterServer {
     implicit val mat: Materializer = ActorMaterializer()
     implicit val ec: ExecutionContext = sys.dispatcher
 
-    val service: HttpRequest => Future[HttpResponse] = GreeterServiceHandler(new GreeterServiceImpl(mat))
+    val service = GreeterServiceHandler(new GreeterServiceImpl(mat))
+      .orElse { case _  => Future.successful(HttpResponse(StatusCodes.NotFound)) }
```
Comment (chbatey) [inline plugin-tester-scala/src/main/scala/io/grpc/examples/helloworld/GreeterServer.scala]: This previously passed a PartialFunction to akka http meaning that an unmatched route blew up with a MatchError

**Comment by raboof** [comment]
Hmm, compile error in the plugin tester:
```
[error] /home/travis/build/akka/akka-grpc/plugin-tester-scala/src/main/scala/io/grpc/examples/helloworld/GreeterServer.scala:39:15: missing parameter type for expanded function
[error] The argument types of an anonymous function must be fully known. (SLS 8.5)
[error] Expected type was: PartialFunction[?,?]
[error]       .orElse { case _  => Future.successful(HttpResponse(StatusCodes.NotFound)) }
[error]               ^
[error] one error found
```

**Comment by chbatey** [comment]
Doh, fixed now

**Pull request merged**